packages/react-vapor/src/components/editor/CodeEditor.tsx

```diff
@@ -81,6 +82,7 @@ export class CodeEditor extends React.Component<ICodeEditorProps, CodeEditorStat
                 onChange={(editor, data, value: string) => this.props.onChange?.(value)}
                 options={{
                     ...CodeEditor.defaultOptions,
+                    lineWrapping: this.props.lineWrapping,

```

Comment:
Wasn't already available through `this.props.options`? 🤔

Reply:
not for the json editor

Reply:
Oh I see, then perhaps we could have added the options prop to the JSONEditor instead. That way it would've stayed uniform and scaled better if we need other options than line wrapping.

Reply:
yeah i could do that, but I didn't know if we wanted to expose the whole editor configuration. Otherwise I can just expose linewrapping on jsoneditor and then pass in options object with linewrapping flag set to whatever user specifies to code editor. I was trying to make a quick fix for another pr in admin-ui :sweat_smile:

Reply:
Its fine as it is 👍  I was just thinking "out loud" 🙂